DEV-GUIDE.md

```diff
@@ -146,14 +146,14 @@ export KUBECONFIG=${HOME}/.kube/config
 
 ```
 
-Execute a one time operation to contain the Istio source trees.
+Execute once, to create a directory for the Istio source trees.
 
 ```shell
 mkdir -p $ISTIO
 ```
 
-As the steps recommmended in this section change both the user's groups
-information as well as the $PATH and environment, please logout of the
+As the steps recommended in this section change both the user's groups
+information as well as the $PATH and environment, please log out of the
```

Comment:
let’s just delete that paragraph, there is no group change and no need to logout

Reply:
Correct - updating to remove reference to groups. There are still changes to environment and PATH which require a log in.

Reply:
no it requires to source or set the path, I don't think we should ask people to logout to change the path it sounds amateurish